src/modules/pageDataGroups.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import ClassNames from 'classnames';
+import React from 'react';
+
+const PageDataGroups = (props) => {
```

Comment:
Let's change this to a basic `function`.

Reply:
Fixed